### PR TITLE
Fix release todo issue template

### PR DESCRIPTION
**Pull request by kselena**
<h1 style="font-weight:bold" align="center">Preview Feature Pull Request</h1>
<h2 style="font-weight:bold" align="center">✅Code Review Checklist✅</h2>

- [x] The **_head(source)_** branch for this pull request is a **_preview feature_** branch, with the correct naming syntax, in the following example:
  - A **_head(source)_** branch is the branch being merged into a target branch.
    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/feature/<issue-num>-<description>
      Example: preview/feature/123-my-preview-feature
      ```
    </details>

- [x] The **_head(source)_** branch for this pull request is created from a **_preview release_** branch, with the correct naming syntax, in the following example:
  <details closed><summary>Example</summary>

    ``` xml
    Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
    Example: preview/v1.2.3-preview.4
    ```
  </details>

- [x] The **_base(target)_** branch for this pull request is a **_preview release_** branch, with the correct naming syntax, in the following example:
  - A **_base(target)_** branch is the branch that the **_head(source)_** branch is merging into.

    <details closed><summary>Example</summary>

      ``` xml
      Syntax: preview/v<major>.<minor>.<patch>-preview.<prev-num>
      Example: preview/v1.2.3-preview.4
      ```
    </details>

💡For more information on branching, refer to the projects [branching documentation](../../Documentation/Branching.md).

- [x] Pull request title matches the title of the associated issue.

- [x] Associated issue exists and is linked to this pull request.
  - One issue per pull request.

- [x] The labels attached to this PR match the labels attached to the associated issue.

- [x] My code follows the coding style of this project.
  - The style is enforced by the **_.editorconfig_** files in the project and displayed as warnings.  If there is an edge case, reach out and let's discuss it.

- [x] A preview-label label has been added to the pull request.

- [x] I have manually tested my code changes to the best of my ability.

- [x] All tests passed locally.
  - This is required because unit tests might pass locally but not in the CI environment during the status check process or vice-versa.
  - Tests might pass on the developer's machine but not necessarily on the code reviewer's machine.
  - This does not mean that the tests pass in both the local and CI environment.
  - Status checks run for this pull request when merging into the **_preview/v\*.\*.\*-preview.\*_** branch.  These status checks run every time a change is pushed to the pull request.  These checks validate version syntax, tagging, builds, unit tests, and more.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/release-todo-issue-template.yml [hunk 1]
```diff
@@ -55,9 +57,9 @@ body:
       
       | Release Types         | Label Used                                                                             |
       |-----------------------|----------------------------------------------------------------------------------------|
-      | Preview Release       | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Preview%20Release         |
-      | Production Release    | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Production%20Release      |
-      | Hot Fix Release       | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Hot%20Fix%20Release       |
+      | Preview Release       | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Preview%20Release         |
+      | Production Release    | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Production%20Release      |
+      | Hot Fix Release       | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Hot%20Fix%20Release       |
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/release-todo-issue-template.yml:62]: Please fix URL's.  They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items.

```suggestion
      | Preview Release       | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Preview%20Release         |
      | Production Release    | https://github.com/KinsonDigital/[TODO: add project name here]/labels/%F0%9F%9A%80Production%20Release      |
      | Hot Fix Release       | https://github.com/KinsonDigital/Velaptor/labels/%F0%9F%9A%80Hot%20Fix%20Release       |
```

File: .github/ISSUE_TEMPLATE/release-todo-issue-template.yml [hunk 2]
```diff
@@ -126,8 +114,8 @@ body:
       **_<details closed><summary>Release Execution</summary>_**
 
       The release is performed by running one of the release workflows below:
-      - Preview Release
-      - Production Release
+      - [Preview Release](https://github.com/KinsonDigital/Velaptor/actions/workflows/preview-release.yml)
+      - [Production Release](https://github.com/KinsonDigital/Velaptor/actions/workflows/qa-prod-releases.yml)
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/release-todo-issue-template.yml:118]: Please fix URL's. They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items

```suggestion
      - [Preview Release](https://github.com/KinsonDigital/[TODO: add project name here]/actions/workflows/preview-release.yml)
      - [Production Release](https://github.com/KinsonDigital/[TODO: add project name here]/actions/workflows/qa-prod-releases.yml)
```

File: .github/ISSUE_TEMPLATE/release-todo-issue-template.yml [hunk 3]
```diff
@@ -10,40 +10,42 @@ body:
...
 - type: checkboxes
   attributes:
-    label: Post-Release To Do List
+    label: Post-Release ToDo List
     description: Verify that release went smoothly.
     options:
-    - label: The release displayed in [GitHub Releases](https://github.com/KinsonDigital/[TODO: add project name here]/releases) is correct.    
-    - label: The [nuget package](https://www.nuget.org/packages/KinsonDigital.[TODO: add project name here]/) has been successfully deployed to the public repository.
+    - label: The release displayed in [GitHub Releases](https://github.com/KinsonDigital/Velaptor/releases) is correct.    
+    - label: The [nuget package](https://www.nuget.org/packages/KinsonDigital.Velaptor/) has been successfully deployed to the public repository.
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/release-todo-issue-template.yml:44]: Please fix URL's. They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items.

```suggestion
    - label: The release displayed in [GitHub Releases](https://github.com/KinsonDigital/[TODO: add project name here]/releases) is correct.    
    - label: The [nuget package](https://www.nuget.org/packages/KinsonDigital.[TODO: add project name here]/) has been successfully deployed to the public repository.
```

File: .github/ISSUE_TEMPLATE/release-todo-issue-template.yml [hunk 4]
```diff
@@ -10,40 +10,42 @@ body:
...
+    - label: Announcement of release on [Twitter](https://twitter.com/KDCoder) verified. (Announcement should be performed automatically with the release)
+    - label: Announcement has been pushed to [Ko-Fi](https://ko-fi.com/kinsondigital).
+    - label: Announcement has been pushed to the correct [GitHub Release Discussion Board](https://github.com/KinsonDigital/Velaptor/discussions/categories/releases).
```
Comment (KinsonDigitalAdmin) [inline .github/ISSUE_TEMPLATE/release-todo-issue-template.yml:47]: Please fix URL's. They used to have the TODO instead of the project name, but when you replaced the entire template with Velaptor's, you forgot to fix the URL's to show the todo items.

```suggestion
    - label: Announcement has been pushed to the correct [GitHub Release Discussion Board](https://github.com/KinsonDigital/[TODO: add project name here]/discussions/categories/releases).
```

**Review by KinsonDigitalAdmin — CHANGES_REQUESTED** [review]
The URL's show Velaptor instead of a template repo TODO item.

**Review by KinsonDigitalAdmin — APPROVED** [review]
